### ISSUE TITLE
lost one  result in report.html

### ISSUE THREAD

**Issue by jinyu00** [michenriksen/aquatone#185]
The urls
```
cat ../aquatone_urls.txt
http://cs.nuaa.edu.cn/
http://job.nuaa.edu.cn/
http://aao.nuaa.edu.cn/
http://www.cnblogs.com/hac425/
```
And the command line
```
root@kali:~/share/tmp/nuaa/nuaa# cat ../aquatone_urls.txt | aquatone -screenshot-timeout 100000
aquatone v1.7.0-beta started at 2019-05-14T11:29:36-04:00

Targets    : 4
Threads    : 4
Ports      : 80, 443, 8000, 8080, 8443
Output dir : .

http://www.cnblogs.com/hac425/: 200 OK
http://cs.nuaa.edu.cn/: 200 OK
http://job.nuaa.edu.cn/: 200 OK
http://aao.nuaa.edu.cn/: 200 OK
http://www.cnblogs.com/hac425/: screenshot successful
http://cs.nuaa.edu.cn/: screenshot successful
http://aao.nuaa.edu.cn/: screenshot successful
http://job.nuaa.edu.cn/: screenshot successful
Calculating page structures... done
Clustering similar pages... done
Generating HTML report... done

Writing session file...Time:
 - Started at  : 2019-05-14T11:29:36-04:00
 - Finished at : 2019-05-14T11:30:08-04:00
 - Duration    : 32s

Requests:
 - Successful : 4
 - Failed     : 0

 - 2xx : 4
 - 3xx : 0
 - 4xx : 0
 - 5xx : 0

Screenshots:
 - Successful : 4
 - Failed     : 0

Wrote HTML report to: aquatone_report.html 
```

And when I see the report , I find that  aao.nuaa.edu.cn's screenshots is missing

![](https://raw.githubusercontent.com/jinyu00/image/master/blog20190514233506.png)

**Comment by michenriksen** [comment]
Thanks for reporting this and for giving enough information to debug :)

I will take a look at what is happening!

**Comment by michenriksen** [comment]
I identified the problem and released [v1.7.0-beta.2](https://github.com/michenriksen/aquatone/releases/tag/v1.7.0-beta.2) which fixes it.

Thanks again for reporting the issue!
